packages/core/admin/server/controllers/authentication.js

```diff
@@ -149,6 +167,24 @@ module.exports = {
     ctx.status = 204;
   },
 
+  async multiFactorAuthentication(ctx) {
+    const input = ctx.request.body;
+    await validateMultiFactorAuthenticationInput(input);
+    if (input.code !== ctx.session.verificationCode) {
+      // Throw forbidden error if verification code is incorrect
+      ctx.status = 403;
+    } else {
+      // Set token if valid
+      ctx.status = 200;
+      ctx.body = {
```

Comment:
You don't need to manually set the body you can just return the object

Reply:
Do you mean as in 
```
  async multiFactorAuthentication(ctx) {
    const input = ctx.request.body;
    await validateMultiFactorAuthenticationInput(input);
    if (input.code !== ctx.session.verificationCode) {
      // Throw forbidden error if verification code is incorrect
      throw new ForbiddenError("Verification code is incorrect");
    } else {
      // Set token if valid
      return {
        data: {
          token: getService('token').createJwtToken(ctx.session.user),
          rememberMe: ctx.session.rememberMe
        },
      };
    }
  },
```
I was copying how the other functions work in that file like `resetPassword` and `register`. They all set the data into the body and don't return anything.